doc/source/tutorial/ndimage.rst

```diff
@@ -461,7 +461,7 @@ information).
   object). The :func:`generic_filter1d` function iterates over the
   lines of an array and calls :func:`function` at each line. The
   arguments that are passed to :func:`function` are one-dimensional
-  arrays of the :ctype:`tFloat64` type. The first contains the values
+  arrays of the :c:type:`tFloat64` type. The first contains the values
```

Comment:
Here's the left-over numarray remnant that needs to be cleaned up: https://github.com/scipy/scipy/blob/master/scipy/ndimage/src/nd_image.h#L67. I'll open a separate issue for that if it doesn't exist yet.